server.js

```diff
@@ -176,12 +191,15 @@ Game.prototype.makeLink = function(player)
 Game.prototype.sendInvitation = function(player, subject)
 {
     try {
+        const gameLink = this.makeLink(player)
+        const sender = `scrabble@${process.env.MAILGUN_DOMAIN}`;
```

Comment:
Maybe processing the environment configuration into the config object upon startup would be nicer than spreading the conditionals over the code.  This is another place where such a conditional would be needed.